source/documentation/using_the_api/linked_registers.md

```diff
@@ -0,0 +1,18 @@
+## Linked registers
+
+Registers can be linked by CURIEs and foreign keys. 
```

Comment:
Perhaps it is worth mentioning that CURIEs are the newest and preferred mechanism so new registers are expected to only use CURIEs.